### PR TITLE
Support blue Flyouts.

### PR DESCRIPTION
**Pull request by quietgolfer**
I'm supporting blue Flyouts.  Here's a picture.

screen shot 2018-06-08 at 1 30 39 pm

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 7f73c5fbd392cde2eacea92fb09d54c8a095da28

https://deploy-preview-249--gestalt.netlify.com

**Comment by pinterest-ios** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Bundle size increase
  </td>
    </tr>
  </tbody>
</table>



Filename | Size Diff | Gzip Diff | Prev Size | Current Size | Prev Gzip | Current Gzip
--- | --- | --- | --- | --- | --- | --- |
gestalt.js | :small_red_triangle: 0.0% | :small_red_triangle_down: 0.0% | 268.27 KB | 268.29 KB | 59.54 KB | 59.53 KB
gestalt.es.js | :small_red_triangle: 0.0% | :small_red_triangle_down: 0.0% | 265.64 KB | 265.67 KB | 59.18 KB | 59.17 KB
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by quietgolfer** [comment]
Yes, Scott says it's PDS approved.

I don't know how we should handle the focus.  Maybe have a different prop for that?

File: packages/gestalt/src/Flyout/__tests__/Flyout.jsdom.test.js [hunk 1]
```diff
@@ -38,6 +38,24 @@ test('Flyout renders as error', () => {
   expect(tree).toMatchSnapshot();
 });
 
+test('Flyout renders as blue', () => {
```
Comment (chrislloyd) [inline packages/gestalt/src/Flyout/__tests__/Flyout.jsdom.test.js]: thank you 🙏

**Comment by christianvuerings** [comment]
@pfarejowicz for now we should indeed change it to `shouldFocus={color !== 'orange'}`.

It is odd to do this depending on the color. Moving forward we should move to inline errors instead (not using the flyout) so we can remove this `shouldFocus` check and always set focus on the flyout.

**Comment by quietgolfer** [comment]
I changed the onFocus to "!== orange".

**Comment by quietgolfer** [comment]
I'm getting the following error.

ERROR: denied: The repository with name 'pinterest' in registry with id '053722134931' already has the maximum allowed number of images which is '1000'
--
  | 🚨 Error: The plugin github.com/buildkite-plugins/docker-compose-buildkite-plugin command hook exited with an error: Error running `/bin/bash -c /tmp/buildkite-agent-bootstrap-hook-runner-915632660`: exit status 1

**Pull request merged**